official/resnet/resnet_run_loop.py

```diff
@@ -196,6 +198,7 @@ def resnet_model_fn(features, labels, mode, model_class,
       If set to None, the format is dependent on whether a GPU is available.
     version: Integer representing which version of the ResNet network to use.
       See README for details. Valid values: [1, 2]
+    loss_scale: The factor to scale the loss for numerical stability.
```

Comment:
Please describe further, even if just to say, see the arg parser.

Reply:
Done.